RELEASE.md

```diff
@@ -13,7 +13,7 @@ the `hab-launch` binary has a separate release process. See
 
 Start with the [autogenerated CHANGELOG](https://github.com/habitat-sh/habitat/blob/master/CHANGELOG.md) - Copy and paste this into a google doc for collaboration.
 
-Look through the merged pull requests and see if any of them is marked with the [C-breaking] label - if they are, make sure to mark their entry in the release notes with [BREAKING CHANGE].
+Look through the merged pull requests and see if any of them is marked with the [T-breaking] label - if they are, make sure to mark their entry in the release notes with [BREAKING CHANGE].
```

Comment:
Looks like this is old process; now, when we add`X-change` to any PR, Expeditor will automatically add an entry to the `Behavioral Changes` section of the changelog. You can just look at that section without having to do any kind of manual filtering at all. Which is cool.

I think we need to revisit some of the labels we have; some seem superfluous now.